docs/userguide/periodic-tasks.rst

```diff
@@ -273,8 +273,8 @@ Some examples:
 | ``crontab(0, 0, day_of_month='11',``    | Execute on the eleventh of May every year. |
 |          ``month_of_year='5')``         |                                            |
 +-----------------------------------------+--------------------------------------------+
-| ``crontab(0, 0,``                       | Execute on the first month of every        |
-|         ``month_of_year='*/3')``        | quarter.                                   |
+| ``crontab(0, 0,``                       | Execute each day(!) on the first month     |
```

Comment:
Is there a need for the exclamation mark?

Reply:
I think `each` should be replaced by `every`.